README.md

```diff
@@ -16,18 +16,21 @@ This cookbook is maintained by the Sous Chefs. The Sous Chefs are a community of
 
 ### chef-client
 
-- Requires chef-client 12.15 and above.
+- Requires chef-client 15.5 and above.
 
 ### Platforms
 
-Tested on the platforms below with distro installs and upstream Varnish packaging versions 3.0, 4.0, 4.1, 5, 5.1, 5.2, 6.0 and 6.1 unless otherwise noted.
+Tested on the platforms below with distro installs and upstream Varnish packaging versions 6.6 and 7.0 unless otherwise noted.
 
-| Varnish      |  3.0  |  4.0  |  4.1  |   5   |  5.1  |  5.2  |  6.0  |  6.1  | distro |
```

Comment:
Do we really need this test matrix in markdown? Seems like a lot of effort to keep up to date?

Reply:
Yeah we can remove this.